assets/src/edit-story/constants/media.js

```diff
@@ -17,3 +17,25 @@
 const GB_IN_BYTES = 1024 * 1024 * 1024;
 
 export const MEDIA_TRANSCODING_MAX_FILE_SIZE = 2 * GB_IN_BYTES;
+
+// List of mime types supported by the video transcoding.
+// Based on the demuxers list as of FFmpeg 4.3.
+export const MEDIA_TRANSCODING_SUPPORTED_INPUT_TYPES = [
```

Comment:
Should this be a variable in PHP with a filter like `get_allowed_image_mime_types`

https://github.com/google/web-stories-wp/blob/3438dca71e1b604111f70404e806207f9c7a21cf/includes/Traits/Types.php#L112-L140

So `get_allowed_transcodable_video_mime_types`

Reply:
What would be a good reason to offer such a filter?

This list is based on what ffmpeg actually supports. So one cannot simply _add_ more types to list, as it doesn't work / make sense.

And I don't see why someone would want to _remove_ something from the list.

Reply:
I agree, just checking to see if their was a reason for this.